blog/urls.py

```diff
@@ -0,0 +1,22 @@
+"""blog URL Configuration.
+
+The `urlpatterns` list routes URLs to views. For more information please see:
```

Comment:
Again, this is useful when you first start working with django, but then it's a pointless noise you don't want to keep in your project.